### PR TITLE
Define Angular Services for Wallet Account Backup

### PR DESCRIPTION
**Pull request by olehyev138**
Back up component created.

Accounts form selection component created for representing the selection of the accounts in the Backup and Exit of the accounts

added button on the actions for navigating to the backup page

### CODE REVIEW

**Comment by rauljordan** [comment]
We definitely need to find a solution or an alternative approach, this UI does not look good at all. However, I understand this is more of a task for a designer. We will look into this a bit later.

Screen Shot 2021-03-15 at 5 20 41 PM

There is no need for 3 buttons. We only need Cancel and Submit, with standard spacing between them
Screen Shot 2021-03-15 at 5 21 09 PM

File: src/app/modules/wallet/pages/account-backup/account-backup.component.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<app-breadcrumb></app-breadcrumb>
+<mat-card class="md:w-2/3">
+    <div class="px-6 pb-4">
+        <div class="import-keys-form">
+            <div class="text-white text-xl mt-4">
+                Backup Selected Account(s)
```
Comment (rkapka) [inline src/app/modules/wallet/pages/account-backup/account-backup.component.html:6]:
```suggestion
                Back Up Selected Account(s)
```

File: src/app/modules/wallet/components/account-actions/account-actions.component.html [hunk 2]
```diff
@@ -21,8 +21,18 @@
       <mat-icon>exit_to_app</mat-icon>
     </button>
   </a>
+  <a class="ml-3"
+    [routerLink]="[ '/dashboard', 'wallet','accounts','backup']">
+    <button mat-raised-button
+      color="accent"
+      class="large-btn ml-1">
+      Back up accounts
```
Comment (rkapka) [inline src/app/modules/wallet/components/account-actions/account-actions.component.html:29]:
```suggestion
      Back Up Accounts
```
For consistency with other buttons.

File: src/app/modules/wallet/pages/account-backup/account-backup.component.ts [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  ): Observable<BackupAccountsResponse> {
+    return this.walletService.backUpAccounts(request).pipe(
+      tap(() => {
+        this.notificationService.notifySuccess(
+          `Successfully backed up ${request.publicKeys.length} accounts`
+        );
+      }),
+      catchError((err) => {
+        this.notificationService.notifyError(
+          'Error happened while backing up the accounts'
```
Comment (rkapka) [inline src/app/modules/wallet/pages/account-backup/account-backup.component.ts:84]:
```suggestion
          'An error occurred during backup'
```

**Review by rkapka — CHANGES_REQUESTED** [review]
I can click the button without selecting any accounts. It should not be possible to back up 0 accounts.

File: src/app/modules/wallet/pages/account-backup/account-backup.component.ts (hunk 3 above)
Comment (olehyev138) [reply src/app/modules/wallet/pages/account-backup/account-backup.component.ts]: all fixed

**Review by rkapka — APPROVED** [review]

**Pull request merged**